cardano-connector/module/src/CardanoService.ts

```diff
@@ -117,23 +117,29 @@ export abstract class CardanoService {
     const graphQLUrl = await this.getGraphQLEndpoint();
     const { transactions } = (
       await axios.post(graphQLUrl, {
-        query: `{ transactions (where: { inputs: {address:{_eq: "${address}" }}}) {
-          block { hash number }
-          blockIndex
-          deposit
-          fee
-          inputs { address sourceTxHash sourceTxIndex }
-          inputs_aggregate { aggregate { count } }
-          outputs { address index txHash value }
-          outputs_aggregate { aggregate { count }}
-          invalidBefore
-          invalidHereafter
-          size
-          totalOutput
-          includedAt
-          withdrawals { address amount transaction { hash }}
-          withdrawals_aggregate { aggregate { count } }
-        } }`,
+        query: `{ transactions (where: {
+            _or: [
+              { inputs: { address: { _eq: "${address}" } } }
+              { outputs: { address: { _eq: "${address}" } } }
```

Comment:
When in doubt, check the Cardano Explorer:
https://explorer.cardano-testnet.iohkdev.io/en/address?address=addr_test1vrrvmd9yuhgzjzgav6v3gj5csx6f7cw65ptc2dm42tl437sy60yzw

Where they count both inputs and outputs for an address.